[DOCS] allow_duplicates option for append processor (#62336)
elastic/elasticsearch: docs/reference/ingest/processors/append.asciidoc

| Name      | Required  | Default  | Description
| `field`  | yes       | -        | The field to be appended to. Supports <<accessing-template-fields,template snippets>>.
| `value`  | yes       | -        | The value to be appended. Supports <<accessing-template-fields,template snippets>>.

> | `allow_duplicates` | no | true  | When false, ignores any of the values to be
>                                     appended that are already present in the
>                                     target field.

Reviewer: Just a reword for clarity.
```suggestion
| `allow_duplicates` | no | true  | If `false`, the processor does not append
values already present in the field.
```
